docs/de/contribute/README.md

```diff
@@ -0,0 +1,128 @@
+
+Klicke auf den Bereich, der dich interessiert:
+
+- [Spenden](#spenden)
+- [Unterstütze dein Netzwerk](#unterstutze-dein-netzwerk)
+- [Betreibe ein eigenes Netzwerk](#betreibe-ein-eigenes-netzwerk)
+- [Entwickle und unterstütze die Software](#entwickle-und-unterstutze-die-software)
+- [Weitere Ideen zum Mitmachen?](#weitere-ideen-zum-mitmachen)
+
+### Spenden
+
+Die Software des Netzwerks ist frei verfügbar – Betrieb, Unterhalt und Weiterentwicklung kosten hingegen Geld.
+
+Der Verein [busFaktor() e.V.](https://busfaktor.org/de/) koordiniert und fördert die Programmierung von *ocelot.social*.
+Unterstütze die Nachhaltigkeit  der Software und deren Netzwerke gerne mit einer Einmal- oder Dauerspende:
+
+<a href="https://busfaktor.org/de/spenden" target="_blank">
+  <Button class="donate-button">
+    Hier spenden
+  </Button>
+</a>
+
+### Unterstütze dein Netzwerk
+
+#### Nutze die aktiven Netzwerke
+
+Sei mit dabei und werde Nutzer eines Netzwerks, das mit unserer Software läuft.
+
+- Registriere dich als Nutzer.
+- Ergänze dein Benutzerprofil mit einem Profilbild, deinem ungefähren Standort und weiteren Angaben über dich.
+- Schreibe lesenswerte und lesefreundliche Beiträge. Ergänze sie mit einem Bild und mit Hashtags.
+- Kommentiere die Beiträge anderer Nutzer.
+- Empfehle lesenswerte Beiträge.
+- Folge Nutzern.
+- Gründe eine Gruppe und lade Mitglieder ein.
+- Trage dazu bei, dass dein Netzwerk ein angenehmer und interessanter Ort für dich und deine Freunde ist.
+
+#### Belebe dein Netzwerk
+
+Bist du Mitglied eines Netzwerks, dann möchtest du dort interessanten Menschen begegnen.
+Mit mehr Nutzern wird dein Netzwerk dynamisch und vielfältig.
+
+- Erzähle deinen Freunden und Verwandten vom Netzwerk.
+- Lade sie ins Netzwerk ein (→ [Anleitung im Wiki](https://github.com/Ocelot-Social-Community/Ocelot-Social/wiki/de:Invitations)).
+- Veröffentliche den Link zum Netzwerk auf deiner Website, deinem Blog oder in anderen sozialen Netzwerken.
+- Berichte vor Publikum vom Netzwerk.
+
+#### Weitere Möglichkeiten dich zu engagieren
+
+Viele Netzwerke sind Mitmach-Netzwerke und sind dankbar für deine Hilfe – vielleicht als Moderator?
+Für weitere Informationen besuche die Website des Betreibers oder kontaktiere ihn.
+
+### Betreibe ein eigenes Netzwerk
+
+Mit der freien Software *ocelot.social* und dem nötigen Know-How kannst du ein eigenes Netzwerk betreiben (→ [Dokumentation](https://docs.ocelot.social/deployment/), auf Englisch).
+
+Benötigst du dabei Hilfe, dann [unterstützen](/de/contact/) dich unsere Entwickler gerne.
+
+### Entwickle und unterstütze die Software
+
+#### Mach die *ocelot.social*-Software bekannt
+
+*›Spread the word!‹*
+
+Ob Verein, Bewegung oder Institution – *ocelot.social* kann als soziales Netzwerk für viele Arten von Gruppen und Organisationen eingesetzt werden.
+
+- Veröffentliche den Link zu *ocelot.social* auf deiner Website, deinem Blog oder in sozialen Netzwerken.
```

Comment:
-Veröffentliche den Link zu *[ocelot.social](https://ocelot.social/)* auf deiner Website, deinem Blog oder in sozialen Netzwerken.